MAINTAIN.md

```diff
@@ -148,8 +148,6 @@ These dependencies are "vendored" (inlined), we must update the sources manually
 * `src/xdiff/`: [xdiff](https://github.com/git/git/tree/master/xdiff)
 * `src/cjson/`: [lua-cjson](https://github.com/openresty/lua-cjson)
 * `src/klib/`: [Klib](https://github.com/attractivechaos/klib)
-* `src/vterm/`: [libvterm](https://www.leonerd.org.uk/code/libvterm/),
```

Comment:
are we not planning to look at upstream ever?

Reply:
Already discussed here: https://github.com/neovim/neovim/pull/30047#discussion_r1740455806

I can re-add it if you disagree with the thought process.